code/modules/antagonists/abductor/machinery/camera.dm

```diff
@@ -59,16 +59,24 @@
 	return HAS_TRAIT(H, TRAIT_ABDUCTOR_SCIENTIST_TRAINING)
 
 /datum/action/innate/teleport_in
+///Is the amount of time required between uses
+	var/abductor_pad_cooldown = 8 SECONDS
+///Is used to compare to world.time in order to determine if the action should early return
+	var/on_cooldown
 	name = "Send To"
 	icon_icon = 'icons/mob/actions/actions_minor_antag.dmi'
 	button_icon_state = "beam_down"
 
 /datum/action/innate/teleport_in/Activate()
 	if(!target || !iscarbon(owner))
 		return
+	if(world.time < on_cooldown)
+		to_chat(owner, "<span class='warning'>You must wait for the [target] to cool down before using it again!</span>")
```

Comment:
if you want to have it display the seconds for the cooldown again, you can do this btw
`to_chat(owner, "<span class='warning'>You must wait [DisplayTimeText(on_cooldown - world.time)] to use the [target] again!</span>")`